Todo-script.js

```diff
@@ -1,47 +1,52 @@
-// Dodavanje "close" dugmeta za svaku stavku liste
-const SPAN_CLOSE = "<span class='close'>\u00D7</span>";
-var variable = $("#userInput");
-
-// Ciljanje <ul> preko klase "todo-list"
-var todoList = $(".todo-list");
-
-$("li").each(function () {
-  $(this).append(SPAN_CLOSE);
-});
-
-// Klik na "close" dugme skriva trenutnu stavku
-todoList.on("click", ".close", function () {
-  $(this).parent().css("display", "none");
-});
-
-// Klik na stavku označava je kao "checked"
-todoList.on("click", "li", function () {
-  $(this).toggleClass("checked");
-});
-
-// Dodavanje nove stavke na listu
-$(".add-btn").click(function () {
-  var inputValue = variable.val();
-  if (!inputValue) {
-    showError("You must enter something on To-do list.");
-  } else {
-    var listItem = $("<li>").text(inputValue).append(SPAN_CLOSE);
-    todoList.append(listItem);
-    variable.val("");
+window.onload = function () {
+  // Dodajte svoj kod koji želite da se izvrši nakon učitavanja stranice ovdje
+  const SPAN_CLOSE = "<span class='close'>\u00D7</span>";
+  const POPOUP_TIME = 1500;
+  const POPUP_ERROR = "<div class='error-popup'>";
+  const ERROR_MESSAGE = "You must enter something on To-do list";
+
+  var user_input = $("#userInput");
+
+  // Ciljanje <ul> preko klase "todo-list"
+  var todoList = $(".todo-list");
+  $("li").each(function () {
+    $(this).append(SPAN_CLOSE);
```

Comment:
to avoid having issues with what is referenced with this, and have future problem it is better to use an arrow function:
`
$("li").each(liElement => liElement.append(SPAN_CLOSE));
`

Reply:
This also reflects all other functions in the code, not jsut this one